wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```
@@ -334,52 +334,72 @@
-      }
-    }
-    
+        // Collection camp start date and end date validation
+        if (['afformCollectionCampIntentDetails', 'afformGoonjActivitiesIndividualIntentForm'].includes(ctrl.getFormMeta().name)) {
+          var startDateValue = $element.find("af-field[name='Collection_Camp_Intent_Details.Start_Date'] .crm-form-date-wrapper input.crm-form-date").val();
+          var endDateValue = $element.find("af-field[name='Collection_Camp_Intent_Details.End_Date'] .crm-form-date-wrapper input.crm-form-date").val();
+          var activitiesStartDateValue = $element.find("af-field[name='Goonj_Activities.Start_Date'] .crm-form-date-wrapper input.crm-form-date").val();
+          var activitiesendDateValue = $element.find("af-field[name='Goonj_Activities.End_Date'] .crm-form-date-wrapper input.crm-form-date").val();
+          
+          if (startDateValue && endDateValue || activitiesStartDateValue && activitiesendDateValue ) {
+            var today = new Date();
```

Biome (1.9.4)

[error] Shouldn't redeclare 'today'. Consider to delete it or rename it.

'today' is defined here:


(lint/suspicious/noRedeclare)